build/python.prf

```diff
@@ -3,15 +3,15 @@
 
 # Change this variable to your python version (2.6, 2.7, 3.3, ...)
 isEmpty( PYTHON_VERSION ) {
-  PYTHON_VERSION=$$(PYTHON_VERSION)
+  PYTHON_VERSION=$${PYTHON_VERSION}
```

Comment:
I believe this is wrong. According to the qmake documentation (https://doc.qt.io/qt-5/qmake-language.html) $$() is used to access environment variables, and $${} to access qmake variable ($${PYTHON_VERSION} is almost the same as $$PYTHON_VERSION, but allowing concatenation). And assigning the variable to itself doesn't make sense.